src/main/java/carRacing/view/ResultView.java

```diff
@@ -18,11 +18,26 @@ public static void printRacing(List<RacingCar> records){
     }
 
     private static void printPlayCar(RacingCar car){
-        StringBuilder result = new StringBuilder();
 
-        for (int i = 0; i < car.positionOfRacingCar(); i++) {
-            result.append("-");
+        System.out.println("-".repeat(Math.max(0, car.positionOfRacingCar())));
+    }
+
+    public static void printRacingCarName(List<RacingCar> records){
+        for (RacingCar racingCar: records) {
+            printPlayCarName(racingCar);
         }
-        System.out.println(result);
+        System.out.println();
+    }
+
+    private static void printPlayCarName(RacingCar car){
+
+        System.out.println(car.NameOfCar() + " : " + "-"
+                .repeat(Math.max(0, car.positionOfRacingCar())));
+    }
+
+    public static void WinnerOfRacing(List<String> name){
```

Comment:
```suggestion
    public static void winnerOfRacing(WinnerNames names) {
```
값을 꺼내서 주지말고 객체 그대로 전달하면 더 좋을 것 같아요